.ci/docker/common/install_onnx.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
Can ONNX exporter team have the access to change this file?https://github.com/pytorch/pytorch/blob/e7d795dccd28bfa556ee5b073520c3e00a68aabe/.github/merge_rules.yaml#L1-L4

Reply:
Good point!  I will update the merge rules accordingly.  Thank you for noticing this.

Reply:
Thanks!